Lib/test/test_enum.py

```diff
@@ -173,6 +173,9 @@ class Holiday(date, Enum):
             IDES_OF_MARCH = 2013, 3, 15
         self.Holiday = Holiday
 
+        from http import HTTPStatus
+        self.HTTPStatus = HTTPStatus
```

Comment:
This test should be in the test for the *http* module.    Also, the test is too brittle and would immediately break if a new code is added or if the API for ints were to be expanded.

Consider something like this:

   self.assertTrue( {'value', 'name', 'description', 'phrase'}  <= set(dir(HTTPStatus(404))))

Reply:
Indeed my tests would break in that case, thanks for your review :)

I've made the following changes:

- The HTTPStatus test is now in test_httplib.py, is it the right place?
- two implemented tests are checking that dir() listing contains attributes

I have made the requested changes; please review again.